DOCUMENTATION.md

```diff
@@ -368,6 +368,68 @@ If the request is not successful, the client returns an `HTTPError` containing t
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification.|
 
 
+### Send a precompiled Letter
```

Comment:
If we want to have 'Send a precompiled letter' show up in the left hand nav, this heading needs to be bigger (`##` instead of `###`). Some of the heading sizes below then need to be adjusted to be made one size bigger too.